ocaml/xapi/xapi_globs.ml

```diff
@@ -468,6 +468,9 @@ let rpu_allowed_vdi_operations = [
   `blocked;
 ]
 
+(* We might consider restricting this further. *)
```

Comment:
Actually I don't think it makes sense to restrict this further, because the remaining operations are quite basic.
But it's harmless to have this comment here.

 Note that it's important to exclude `mirror`, because because the `migrate_send` VM operation was excluded in https://github.com/xapi-project/xen-api/pull/3139 . To fix CA-246335 we disallowed SXM in a partially upgraded pool in that PR, so we should also exclude the corresponding VDI operation, `mirror`, which is used for VDI.pool_migrate.